LondonBibleSociety/BSLet177.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+   schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BSLet177" type="mss">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Saʿātāt with Psalms of David</title>
+            <editor key="JK"/>
+            <editor key="AB" role="generalEditor"/>
+            <funder>Akademie der Wissenschaften in Hamburg</funder>
+         </titleStmt>
+         <editionStmt><p>Cataloguing funded by the AHRC-DFG Project
+            'Demarginalizing medieval Africa: Images, texts, and identity
+            in early Solomonic Ethiopia (1270-1527)', grant ref.
+            nos. AH/V002910/1 - 448410109</p></editionStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <pubPlace>Hamburg</pubPlace>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                  licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+                  <msIdentifier>
+                     <repository ref="INS0306BSL"/>
+                     <idno>BFBS Mss 177</idno>
+                     <altIdentifier>
+                        <idno>BSMS 177</idno>
+                     </altIdentifier>
+                     <altIdentifier>
+                        <idno>Cowley IV</idno>
+                     </altIdentifier>
+                  </msIdentifier>
+               <msContents>
+                  <summary/>
+                  <msItem xml:id="ms_i1">
+                     <title ref="LIT1961Mashaf"/>
+                     <textLang mainLang="gez"/>
```

Comment:
above in title statement you wrote "Sa'atat with Psalms of David", should there then here be two points for msContent?

Reply:
I was thinking that the Psalms of David are an integral part of this Sa'atat

Reply:
I think this is an open research question

Reply:
I agree, and I wouldn't add the Psalter as a msContent

Reply:
To clarify: The Psalter is not to be found from the beginning to the end, but there are different Sa'atat prayers in between. So it is quite well integrated into the manuscript.